projects/control-service/projects/base/src/main/java/com/vmware/taurus/security/SecurityConfiguration.java

```diff
@@ -59,6 +69,22 @@
     private final String customClaimName;
     private final Set<String> authorizedCustomClaimValues;
     private final Set<String> authorizedRoles;
+    public static final String[] ENDPOINTS_TO_IGNORE = {
+          "/",
+          "/data-jobs/v2/api-docs",
+          "/data-jobs/swagger-resources/**",
+//        "/data-jobs/configuration/**",
+          "/data-jobs/swagger-ui.html",
+          "/data-jobs/webjars/**",
+          // There should not be sensitive data in prometheus, and it makes
+          // integration with the monitoring system easier if no auth is necessary.
+          "/data-jobs/debug/prometheus",
+          // TODO: likely /data-jobs/debug is too permissive
+          // but until we can expose them in swagger they are very hard to use with Auth.
+          "/data-jobs/debug/**"};
+    private final String kerberosPrincipal;
+    private final String keytabFileLocation;
+    private final String KERBEROS_AUTH_ENABLED_PROPERTY = "datajobs.security.kerberos.enabled";
```

Comment:
my suggest was to add this constatnt in FeatureFlags so you can also reuse it in [KerberosConfig.java](https://github.com/vmware/versatile-data-kit/pull/755/files#diff-ef66e0bc2235c9ce8dabdbd1622fdd3e76efa28513ffcb028c114022f5082d80)

Also it should be `static final` I think.

Reply:
Yes well if it's in the FeatureFlags class the project won't compile - some gimmick of how annotations work I guess or some issue with lombok. It fails with [attribute must be a constant expression error] and if I move the variable to the SecurityConfiguration class it compiles ( I tried with a static final variable from FeatureFlags).